pkg/kratos/service.go

```diff
@@ -43,6 +49,30 @@ type BrowserLocationChangeRequired struct {
 	RedirectTo *string `json:"redirect_to,omitempty"`
 }
 
+type Context struct {
+	Property string `json:"property,omitempty"`
+	Pattern  string `json:"pattern,omitempty"`
+	Reason   string `json:"reason,omitempty"`
+}
+
+type Messages []struct {
+	Id      int    `json:"id,omitempty"`
+	Text    string `json:"text,omitempty"`
+	Type    string `json:"type,omitempty"`
+	Context Context
+}
+
+type UiNode []struct {
+	Messages Messages `json:"messages,omitempty"`
+}
+
+type UiErrorMessages struct {
+	Ui struct {
+		Messages Messages
+		Nodes    UiNode
+	} `json:"ui,omitempty"`
+}
```

Comment:
I think this can be rewritten as
```suggestion
type UiErrorMessages struct {
    Ui kClient.UiContainer `json:"ui"`
}
```